copylot/gui/qt_main_window.py

```diff
@@ -25,6 +34,52 @@ def __init__(self, *args, **kwargs):
         self.setGeometry(self.left, self.top, self.width, self.height)
         self.setDockOptions(QMainWindow.AnimatedDocks | QMainWindow.AllowNestedDocks)
 
+        self.init_defaults = [
```

Comment:
can we also have a file for init defaults and load them? maybe from version to version we might want to push changes to users

Reply:
This won't be in the git directory though, so can't be shipped right? So then the program can't launch initially without the file being placed there?

Reply:
Right now by making it on first launch it is my way of shipping it, but I see what you mean with that being more difficult to change in the future. Not sure how to fix this then.